src/examples/Ext.lean

```diff
@@ -1,19 +1,66 @@
 import challenge_notations
 import challenge_prerequisites
+import for_mathlib.universal_delta_functor.Ext
 
 /-!
 
-This file shows that `Ext` is a universal δ-functor (TODO)
+This file shows that `Ext` is a universal δ-functor
 and performs the computation `Ext^1(ℤ/nℤ, ℤ/nℤ) = ℤ/nℤ`.
 
 -/
 
 noncomputable theory
 
 open category_theory category_theory.limits opposite
+open bounded_homotopy_category bounded_derived_category
+
+section Ext
+
+universes v u
+-- Let's work with an abelian category which has enough projectives.
+variables {A : Type u} [category.{v} A] [abelian A] [enough_projectives A]
+
+/-- This is the (contravariant) delta functor `X ↦ Ext^*(X,B)`. -/
+example (B : A) : Aᵒᵖ ⥤δ Ab.{v} := Ext_δ_functor A B
+
+/-- The `n-th` component of this delta functor. -/
+example (n : ℕ) (X B : A) : Aᵒᵖ ⥤ Ab.{v} := Ext_δ_functor A B n
+example (n : ℕ) (X B : A) : Ext_δ_functor A B n = (Ext' n).flip.obj B := rfl
+
+/-- `Ext' n (X,B) = Ext' n (X[0], B[0])`, functorially in `X`. -/
+example (n : ℕ) (X B : A) : (Ext' n).flip.obj B =
```

Comment:
This isn't using the `X : A` assumption.

Reply:
done